PHP unit fixes

Fredrik82 opened the pull request:
* Added a try-catch clause in `Utils.php` and return false if feeded with bad XML - as dictated in the phpunit test.
* Fixed some typos where the scalar value `false` was misspelled.
* Replaced `assertNotContains` with `assertStringNotContainsString` when asserting strings.

pitbulk commented:
Hi, @Fredrik82

I think the loadXML method should raise an exception if not able to load the XML.

Fredrik82 commented:
> Hi, @Fredrik82
> 
> I think the loadXML method should raise an exception if not able to load the XML.

I agree. I was unsure if the unit-test was supposed to be considered as source of truth or not. I reverted that commit and added a try-catch in the unit test instead.
